apps/ledger-live-desktop/src/renderer/screens/platform/v2/Catalog/index.tsx

```diff
@@ -6,13 +6,27 @@ import { Browse } from "./Browse";
 import { useTranslation } from "react-i18next";
 import { useCatalog, useRecentlyUsedDB } from "../hooks";
 import { LocalLiveAppSection } from "./LocalLiveAppSection";
+import { useLocation } from "react-router-dom";
+import { Categories } from "@ledgerhq/live-common/wallet-api/react";
 
 export function Catalog() {
   const recentlyUsedDB = useRecentlyUsedDB();
 
   const { t } = useTranslation();
-  const { categories, recentlyUsed, disclaimer, search, localLiveApps } =
-    useCatalog(recentlyUsedDB);
+
+  const { state } = useLocation<{
+    category?: Categories["selected"];
+  }>();
+
+  const deeplinkInitialCategory = state?.category;
+
+  const {
+    categories,
+    recentlyUsed,
+    disclaimer,
+    search: searchInput,
+    localLiveApps,
+  } = useCatalog(recentlyUsedDB, deeplinkInitialCategory ?? undefined);
```

Comment:
Do we need the fallback here ?
I expect category to be `Categories["selected"] | undefined`
```suggestion
  } = useCatalog(recentlyUsedDB, deeplinkInitialCategory);
```